perun-base/src/main/java/cz/metacentrum/perun/rpc/serializer/JsonSerializerJSONLITE.java

```diff
@@ -0,0 +1,234 @@
+	}
+
+	@JsonIgnoreProperties({"beanName"})
+	private interface ExceptionMixIn {
+	}
+
+	@JsonIgnoreProperties({"beanName"})
+	private interface CandidateMixIn {
+	}
+
+	@JsonIgnoreProperties({
+		"createdAt", "createdBy", "modifiedAt", "modifiedBy", 
+		"createdByUid", "modifiedByUid", 
+		"beanName"
+		})
+	private interface PerunBeanMixIn {
+	}
+
+	@JsonIgnoreProperties({"beanName"})
+	private interface PerunRequestMixIn {
+	}
+
+	/* FOR Cabinet PerunBeans we need createdBy etc. data */
+	@JsonIgnoreProperties({})
+	private interface CabinetMixIn {
+	}
+
+	@SuppressWarnings("unused")
+	private interface TaskMixIn {
+		@JsonSerialize
+		@JsonProperty(value = "startTime")
+		Long getStartTimeAsLong();
+
+		@JsonIgnore
+		LocalDateTime getStartTime();
+
+		@JsonSerialize
+		@JsonProperty(value = "schedule")
+		Long getScheduleAsLong();
+
+		@JsonIgnore
+		LocalDateTime getSchedule();
+
+		@JsonSerialize
+		@JsonProperty(value = "genEndTime")
+		Long getGenEndTimeAsLong();
+
+		@JsonIgnore
+		LocalDateTime getGenEndTime();
+
+		@JsonSerialize
+		@JsonProperty(value = "sendEndTime")
+		Long getSendEndTimeAsLong();
+
+		@JsonIgnore
+		LocalDateTime getSendEndTime();
+
+		@JsonSerialize
+		@JsonProperty(value = "sendStartTime")
+		Long getSendStartTimeAsLong();
+
+		@JsonIgnore
+		LocalDateTime getSendStartTime();
+
+		@JsonSerialize
+		@JsonProperty(value = "genStartTime")
+		Long getGenStartTimeAsLong();
+
+		@JsonIgnore
+		LocalDateTime getGenStartTime();
+
+		@JsonSerialize
+		@JsonProperty(value = "sentToEngine")
+		Long getSentToEngineAsLong();
+
+		@JsonIgnore
+		LocalDateTime getSentToEngine();
+
+		@JsonSerialize
+		@JsonProperty(value = "endTime")
+		Long getEndTimeAsLong();
+
+		@JsonIgnore
+		LocalDateTime getEndTime();
+	}
+
+	public static final String CONTENT_TYPE = "application/json; charset=utf-8";
+	private static final ObjectMapper mapper = new ObjectMapper();
+	private static final Map<Class<?>,Class<?>> mixinMap = new HashMap<>();
+
+	static {
+		mixinMap.put(Attribute.class, AttributeMixIn.class);
+		mixinMap.put(AttributeDefinition.class, AttributeDefinitionMixIn.class);
+		mixinMap.put(User.class, UserMixIn.class);
+		mixinMap.put(Candidate.class, CandidateMixIn.class);
+		mixinMap.put(PerunException.class, ExceptionMixIn.class);
+		mixinMap.put(PerunRuntimeException.class, ExceptionMixIn.class);
+		mixinMap.put(PerunBean.class, PerunBeanMixIn.class);
+		mixinMap.put(PerunRequest.class, PerunRequestMixIn.class);
+		mixinMap.put(Authorship.class, CabinetMixIn.class);
+		mixinMap.put(Author.class, CabinetMixIn.class);
+		mixinMap.put(Category.class, CabinetMixIn.class);
+		mixinMap.put(Publication.class, CabinetMixIn.class);
+		mixinMap.put(Thanks.class, CabinetMixIn.class);
+		mixinMap.put(Task.class, TaskMixIn.class);
+
+		mapper.setMixIns(mixinMap);
+	}
+
+	private static final JsonFactory jsonFactory = new JsonFactory();
+
+	static {
+		//FIXME odstraneno disable(JsonGenerator.Feature.FLUSH_PASSED_TO_STREAM)
```

Comment:
I know it is a copy, but we should use English here.

Reply:
Updated.